config/manifests/bases/service-binding-operator.clusterserviceversion.yaml

```diff
@@ -20,6 +20,11 @@ spec:
       kind: ServiceBinding
       name: servicebindings.binding.operators.coreos.com
       version: v1alpha1
+    - description: ServiceBinding expresses intent to bind an operator-backed service with an application workload.
```

Comment:
It would be nice that the description contain info that this API is actually service binding specification API

Reply:
What about smething like:
```
Service Binding for Kubernetes. Describes the connection between a Provisioned Service and an Application Projection. Implements Service Binding Specifcation for Kubernetes (https://github.com/k8s-service-bindings/spec).
```